test/test.js

```diff
@@ -42,10 +42,19 @@ describe('Generate APIs and models with WSDLs containing ', function() {
         // check for API/operation signature in generated code
         var index = code.indexOf('StockQuoteServiceStockQuoteBinding.GetQuote = function(GetQuote, callback)'); // eslint-disable-line max-len
         assert.ok(index > -1);
+
         // check for beginning of REST API in generated code
         index = code.indexOf("StockQuoteServiceStockQuoteBinding.remoteMethod('GetQuote',"); // eslint-disable-line max-len
         assert.ok(index > -1);
 
+        // check for utility method that gets JSON value of model at runtime; if available
+        index = code.indexOf('function normalize(parameter) {');
```

Comment:
looks like historically the code coverage for this library has been limited to checking the generated text :) and not running the generated code

Reply:
Yes, I agree. Am I sure they tested manually against a temporary service they served locally. But avoided hosting permanent web services for the CI builds.